http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitorSpec.scala

```diff
@@ -255,7 +255,7 @@ class BaseCloudServiceRuntimeMonitorSpec extends AnyFlatSpec with Matchers with
       // handleCheckTools should have timed out after 10 seconds and the runtime should remain in Deleted status
       elapsed should be >= 10000L
       status shouldBe Some(RuntimeStatus.Deleted)
-      diskStatus shouldBe (Some(DiskStatus.Deleted))
+      diskStatus shouldBe (Some(DiskStatus.Ready))
```

Comment:
For this file and the next, it would be nice to keep the existing test that triggers the disk deletion)

Reply:
Agreed! I'll get a test in there before I request your review again.